src/dashboard/overview/Overview.tsx

```diff
@@ -58,32 +73,54 @@ class Overview extends DashboardContent<PropsType, StateType> {
     return status.status !== "Disabled";
   };
 
+  handleSelectNotifications = () => {
+    console.log("handleSelectNotifications");
+  };
+
   render(): ReactElement {
     const { classes } = this.props;
 
     return (
-      <Grid container spacing={5} className={classes.wrapper}>
-        {this.state.initialLoadCompleted ? (
-          this.state.statuses &&
-          this.state.statuses
-            .filter(this.statusFilter)
-            .sort(
-              (a, b) =>
-                this.sortingOrderByService(a.service) -
-                this.sortingOrderByService(b.service)
-            )
-            .map((status) => (
-              <OverviewItem
-                status={status}
-                key={status.service}
-                opendexdLocked={this.state.opendexdLocked}
-                opendexdNotReady={this.state.opendexdNotReady}
-              ></OverviewItem>
-            ))
-        ) : (
-          <PageLoader />
-        )}
-      </Grid>
+      <div>
+        <div className={classes.headingContainer}>
+          <h1 className={classes.heading}>Overview</h1>
+          <Icon
```

Comment:
Please let's hide it until it has a functionality. Just some false condition here would do it.
Also, thinking of other views, they also need it so would not make sense to keep it in the _Overview_ component but a `PageTitleComponent` or so would be meaningful.

Reply:
Created PageTitle component for it.